models/evolution/model_v0/_package.yml

```diff
@@ -0,0 +1,8 @@
+namespace: EvoTest
+
+cpp:
+  sourcesOutputDir: ../../../cpp/evolution/v0/lib
```

Comment:
In our other models, we have generated to a directory named `generated`. I think  we should be consistent.